src/packages/console/src/pages/loginPage.tsx

```diff
@@ -67,76 +41,110 @@ export const LoginPage = (): React.ReactElement => {
     });
   }
 
-  const onEmailChanged = (event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>): void => {
+  /*const onEmailChanged = (event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>): void => {
     setEmail(event.target.value);
     setEmailError(undefined);
   }
 
   const onPasswordChanged = (event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>): void => {
     setPassword(event.target.value);
     setPasswordError(undefined);
-  }
+  }*/
 
   return (
-    <Container component='main' maxWidth='sm'>
-      <Paper className={classes.paper}>
-        <Typography component='h1' variant='h5'>
-          Log in to everypage
-        </Typography>
-        {isLoading ? (
-          <CircularProgress />
-        ) : (
-          <form className={classes.form} noValidate onSubmit={onLoginClicked}>
-            <TextField
-              variant='outlined'
-              margin='normal'
-              required
-              fullWidth
-              id='email'
-              label='Email Address'
-              name='email'
-              autoComplete='email'
-              autoFocus
-              value={email}
-              onChange={onEmailChanged}
-              error={emailError !== undefined}
-              helperText={emailError}
-            />
-            <TextField
-              variant='outlined'
-              margin='normal'
-              required
-              fullWidth
-              name='password'
-              label='Password'
-              type='password'
-              id='password'
-              autoComplete='current-password'
-              value={password}
-              onChange={onPasswordChanged}
-              error={passwordError !== undefined}
-              helperText={passwordError}
-            />
-            <Button
-              type='submit'
-              fullWidth
-              variant='contained'
-              color='primary'
-              className={classes.submit}
-            >Log In</Button>
-            <Grid container>
-              <Grid item xs>
-                {/* <Link href='#' variant='body2'>
+    <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center', textAlign: "center" }}>
```

Comment:
@krishan711 This the only way I was able to center the box. Please suggest an alternative